native/src/main/java/io/ballerina/lib/ldap/Client.java

```diff
@@ -77,13 +87,22 @@ public final class Client {
     public static final BString PORT = StringUtils.fromString("port");
     public static final BString DOMAIN_NAME = StringUtils.fromString("domainName");
     public static final BString PASSWORD = StringUtils.fromString("password");
+    public static final BString CLIENT_SECURE_SOCKET = StringUtils.fromString("clientSecureSocket");
     public static final String NATIVE_CLIENT = "client";
     public static final String LDAP_RESPONSE = "LdapResponse";
     public static final BString REFERRAL = StringUtils.fromString("referral");
     public static final BString OPERATION_TYPE = StringUtils.fromString("operationType");
     public static final String OBJECT_GUID = "objectGUID";
     public static final String OBJECT_SID = "objectSid";
 
+    //Socket config
+    private static final BString SECURESOCKET_CONFIG_ENABLE_TLS = StringUtils.fromString("enable");
```

Comment:
Can we split the word `SECURESOCKET` into `SECURE_SOCKET`?
```suggestion
    private static final BString SECURE_SOCKET_CONFIG_ENABLE_TLS = StringUtils.fromString("enable");
```